src/mongocxx/exception/private/mongoc_error.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 MongoDB Inc.
+// Copyright 2015 MongoDB Inc.
```

Comment:
Is this just git being confused about that file being removed and this file being separately added?